app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -107,12 +105,24 @@ func NewLoadBalancerSyncer(lbName string, clients map[string]kubeclient.Interfac
 
 // CreateLoadBalancer creates the GCP resources necessary for an L7 GCP load balancer corresponding to the given ingress.
 // clusters is the list of clusters that this load balancer is spread to.
-func (l *LoadBalancerSyncer) CreateLoadBalancer(ing *v1beta1.Ingress, forceUpdate bool, clusters []string) error {
+func (l *LoadBalancerSyncer) CreateLoadBalancer(ing *v1beta1.Ingress, forceUpdate, validate bool, clusters []string) error {
 	client, cErr := getAnyClient(l.clients)
 	if cErr != nil {
 		// No point in continuing without a client.
 		return cErr
 	}
+
+	if validate {
+
+		if err := validations.Validate(l.clients, ing); err != nil {
+			glog.Errorf("Validation failed: %s", err)
```

Comment:
Sorry should have been clearer before.

We do not use glog unless user explicitly uses verbose flag. Hence we dont use glog.Errorf directly. Using glog.V(2).Errorf is fine.
For messages that we want to print without verbosity, we use fmt.Printf().